### PR TITLE
Testing : set file permissions after creating the log (#53)

### LINKED ISSUE #49: Insufficient permissions:
i tried to install shaarli in new server yunohost... but: 
Insufficient permissions:
"data/log.txt" file is not writable

is it a bug ?

### PR DESCRIPTION
**Pull request by lapineige**
Fix #49

Should we add something in the upgrade script to fix current configurations ?

### CODE REVIEW

File: scripts/install [hunk 1]
```diff
@@ -156,6 +147,17 @@ ynh_print_info "Configuring log rotation..."
 touch "$final_path/data/log.txt"
 ynh_use_logrotate "$final_path/data/log.txt"
 
+#=================================================
+# SECURE FILES AND DIRECTORIES
+#=================================================
+# set proper permissions
+ynh_print_info "Set permissions..."
+find $final_path -type f | xargs chmod 644
+find $final_path -type d | xargs chmod 755
```
Comment (alexAubin) [inline scripts/install:156]: Uh I have no idea what I'm talking about but is it really what we wanna do here ?

Is it okay that *all users* on the system will be able to read every file in /var/www/$app ? Are we sure that there's absolutely no info that should stay private in that folder ?

Comment (lapineige) [reply scripts/install:156]: Good point, I don't see any reason to do that.

**Comment by Tagadda** [comment]
Bump! I think this can me merged in a first time;
For now, installation fails without it.

**Comment by lapineige** [comment]
Then we need to assess that file permission issue…

**Pull request merged**